providers/microsoft/azure/src/airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -266,6 +272,14 @@ def check_for_prefix(self, container_name: str, prefix: str, **kwargs) -> bool:
         blobs = self.get_blobs_list(container_name=container_name, prefix=prefix, **kwargs)
         return bool(blobs)
 
+    T = TypeVar("T", AsyncBlobServiceClient, ContainerClient, AsyncContainerClient)
```

Comment:
The TypeVar should be moved to the top level since it is a constant.
Additionally, `BlobServiceClient` should be included in the TypeVar.

```suggestion
    T = TypeVar("T", AsyncBlobServiceClient, BlobServiceClient, ContainerClient, AsyncContainerClient)
```